File: SectorCreator.WorldBuilder/Planet/Planet/WorldBuilderStarport.cs
using SectorCreator.Models.Basic;

namespace SectorCreator.WorldBuilder.Planet.Planet;

public class WorldBuilderStarport
{
    public StarportClass Class { get; set; }
    public bool Highport { get; set; }
    public int DailyTraffic { get; set; }
    public int DailyMaximum { get; set; }
    public int ExpectedWeekly { get; set; }
    public int HighportTotalDockingCapacity { get; set; }
    public int DownportTotalDockingCapacity { get; set; }
    public int ShipyardBuildCapacity { get; set; }
    public int AnnualShipyardOutput { get; set; }
}

File: TravellerCreator/ClientApp/src/app/sector-creator/models/world-builder/world-builder-starport.ts
import {StarportClass} from "./Enums/starport-class";

export class WorldBuilderStarport {
  class: StarportClass;
  highport: boolean;
  dailyTraffic: number;
  dailyMaximum: number;
  expectedWeekly: number;
  highportTotalDockingCapacity: number;
  downportTotalDockingCapacity: number;
  shipyardBuildCapacity: number;
  annualShipyardOutput: number;
}
